tests/samples/songkick/elysianfields.json

```diff
@@ -1,115 +1 @@
ue Aristide Briand", "postalCode": "33152"}, "name": "Le Rocher De Palmer", "sameAs": "http://lerocherdepalmer.fr/", "geo": {"@type": "GeoCoordinates", "latitude": 44.8624327, "longitude": -0.5237826}}, "startDate": "2016-10-27T20:30:00+0200", "performer": [{"@type": "MusicGroup", "name": "Elysian Fields", "sameAs": "http://www.songkick.com/artists/236156-elysian-fields?utm_medium=organic&utm_source=microformat"}, {"@type": "MusicGroup", "name": "Ernst Reijseger", "sameAs": "http://www.songkick.com/artists/13128-ernst-reijseger?utm_medium=organic&utm_source=microformat"}, {"@type": "MusicGroup", "name": "Troy Von Balthazar", "sameAs": "http://www.songkick.com/artists/355304-troy-von-balthazar?utm_medium=organic&utm_source=microformat"}, {"@type": "MusicGroup", "name": "Rachid Taha", "sameAs": "http://www.songkick.com/artists/100389-rachid-taha?utm_medium=organic&utm_source=microformat"}, {"@type": "MusicGroup", "name": "Vaudou Game", "sameAs": "http://www.songkick.com/artists/8195583-vaudou-game?utm_medium=organic&utm_source=microformat"}]}, {"@context": "http://schema.org", "@type": "MusicGroup", "name": "Elysian Fields", "url": "http://www.songkick.com/artists/236156-elysian-fields?utm_medium=organic&utm_source=microformat", "image": "https://images.sk-static.com/images/media/profile_images/artists/236156/card_avatar", "logo": "https://images.sk-static.com/images/media/profile_images/artists/236156/card_avatar", "interactionCount": "5555 UserLikes"}], "opengraph": [{"namespace": {"og": "http://ogp.me/ns#", "fb": "http://www.facebook.com/2008/fbml", "concerts": "http://ogp.me/ns/fb/songkick-concerts#"}, "properties": [["fb:app_id", "308540029359"], ["og:site_name", "Songkick"], ["og:type", "songkick-concerts:artist"], ["og:title", "Elysian Fields"], ["og:description", "Buy tickets for an upcoming Elysian Fields concert near you. List of all Elysian Fields tickets and tour dates for 2017."], ["og:url", "http://www.songkick.com/artists/236156-elysian-fields"], ["og:image", "http://images.sk-static.com/images/media/img/col4/20100330-103600-169450.jpg"]]}], "microformat": [], "rdfa": [{"@id": "http://www.songkick.com/artists/236156-elysian-fields", "al:ios:app_name": [{"@value": "Songkick Concerts"}], "al:ios:app_store_id": [{"@value": "438690886"}], "al:ios:url": [{"@value": "songkick://artists/236156-elysian-fields"}], "http://ogp.me/ns#description": [{"@value": "Buy tickets for an upcoming Elysian Fields concert near you. List of all Elysian Fields tickets and tour dates for 2017."}], "http://ogp.me/ns#image": [{"@value": "http://images.sk-static.com/images/media/img/col4/20100330-103600-169450.jpg"}], "http://ogp.me/ns#site_name": [{"@value": "Songkick"}], "http://ogp.me/ns#title": [{"@value": "Elysian Fields"}], "http://ogp.me/ns#type": [{"@value": "songkick-concerts:artist"}], "http://ogp.me/ns#url": [{"@value": "http://www.songkick.com/artists/236156-elysian-fields"}], "http://www.facebook.com/2008/fbmlapp_id": [{"@value": "308540029359"}]}]}
```

Comment:
Why is indentation removed? It is hard to check a diff without indentation, and it is also hard to read a file.

Reply:
I had to modify these files to include new syntaxes and did not think to keep indentation